deploy/scripts/src/mobile/authorize.test.ts

```diff
@@ -22,6 +22,20 @@ const profiles: ProfileList = {
       ],
       exec: 'authorize'
     }
+  },
+  load: {
+    startJourney: {
+      executor: 'ramping-arrival-rate',
+      startRate: 0,
+      timeUnit: '1s',
+      preAllocatedVUs: 50, // Calculation: 100 journeys / second * 0.5 seconds average journey time
+      maxVUs: 200, // Calculation: 100 journeys / second * 2 seconds maximum journey time
```

Comment:
I'd recommend bumping this up to 300, so it can support just over the max expected 2.5 second response time (from NFRs). Hopefully won't be needed though